app/assets/javascripts/discourse-common/addon/lib/auto-groups.js

```diff
@@ -0,0 +1,56 @@
+export const AUTO_GROUPS = {
```

Comment:
We should avoid generating this by hand since it may mean this file potentially becoming out of sync with the server side data. 

There is a pattern in core that we can use this avoid this problem:

https://github.com/discourse/discourse/blob/5e395d4382f45604b0ac8fc7f1b9fe08b581cb7d/lib/tasks/javascript.rake#L133

Reply:
Yes I was wondering about that, will look into that :+1: